src/App.vue

```diff
@@ -163,23 +162,16 @@ export default {
       )
     },
     shouldDisplayHolochainConnecting () {
-      return this.conductorDisconnected && !this.firstConnect && !this.isChaperoneDisconnected
+      return !isHoloHosted() && this.conductorDisconnected
```

Comment:
In this PR I believe that `Connecting...` will appear on first holochain connect, and `Reconnecting in X` will appear in later ones. Is that not the desired behavior?

Reply:
To be clear, that's because [`reconnectingIn`'s starting value](https://github.com/holochain/elemental-chat-ui/pull/123/files#diff-7c5ef0f6902bd15e89cf0d5d2381603662e19aba2bb6f30264abfe3a69313751L230-R62) causes [`Connecting...` to appear](https://github.com/holochain/elemental-chat-ui/pull/123/files#diff-7a7a37b12ee1265d7e27577ec286120d2cbc0940908635e264a2be44ccb9a8a0L79-L84)

Oh wait. It's starting value got changed to null instead of 0. Okay I'll change that back.